.eslintrc

```diff
@@ -186,7 +185,7 @@
       "always"
     ],
     "sort-vars": 0,
-    "keyword-spacing": [ 2 ],
```

Comment:
This rule was introduced in eslint 2, which is the version of eslint in our package.json. If this rule is throwing warnings for you you'll need to update your version of eslint.